test/integration/api/proxy_rules_controller_test.rb

```diff
@@ -5,138 +5,81 @@
 class Api::ProxyRulesControllerTest < ActionDispatch::IntegrationTest
   def setup
     @provider = FactoryBot.create(:provider_account)
+    @service = FactoryBot.create(:service, account: @provider)
     login_provider @provider
```

Comment:
minor suggestion, since we are here: 🙈  
```suggestion
    login! @provider
```

The whole file is good tho 👍🏼 😉